python/tskit/trees.py

```diff
@@ -6560,28 +6560,32 @@ def diversity(
         What is computed depends on ``mode``:
 
         "site"
-            Mean pairwise genetic diversity: the average across distinct,
-            randomly chosen pairs of chromosomes, of the density of sites at
+            Mean pairwise genetic diversity: the average across every possible pair of
```

Comment:
What if we said "the average over all n choose 2 pairs of sample nodes" or something? This would make it more obvious what we're doing do me

Reply:
That works too.